package.json

```diff
@@ -14,17 +14,17 @@
     "cookie-parser": "1.4.3",
     "express": "4.14.0",
     "express-session": "1.14.2",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "resolve": "1.1.7"
+    "infusion": "2.0.0-dev.20161219T170555Z.5778f7e",
```

Comment:
This version of Infusion has now been deprecated as a result of its faulty version number. I've just rolled 3.0.0-dev.20170127T130413Z.103de6e which includes your gpii-binder transforms, for which cheers.